package.json

```diff
@@ -2,7 +2,7 @@
   "name": "openbci-sdk",
   "version": "0.1.0",
```

Comment:
Yes

--------------------------------------------
 
AJ Keller
Founder & CEO
Push The World LLC
Work: (843) 471-0039
Cell: (203) 733-4538

Warning: The information transmitted is intended only for the person or entity to which it is addressed and may contain confidential and/or privileged material. Any review, retransmission, dissemination or other use of, or taking of any action in reliance upon, this information by persons or entities other than the intended recipient is prohibited. If you received this in error, please contact the sender and destroy this material.
 

> On Dec 31, 2015, at 2:10 PM, Teon L Brooks <notifications@github.com> wrote:
> 
> In package.json:
> 
> > @@ -2,7 +2,7 @@
> >    "name": "openbci-sdk",
> >    "version": "0.1.0",
> does this need to be incremented?
> 
> —
> Reply to this email directly or view it on GitHub.
>